examples/analysis/ubet/examples/ub.cpp

```diff
@@ -0,0 +1,51 @@
+#include <numeric>
```

Comment:
Are we actually using this anywhere? If not, we could consider removing it (and run.sh/runub.sh). Maybe good to keep as a minimalistic exampel?

Reply:
I was keeping them for purposes of having a minimal example, but we didn't end up using them for more than a learning step iirc, so will remove.